cli/.js/src/main/scala/aqua/ArgOpts.scala

```diff
@@ -135,26 +136,26 @@ object ArgOpts {
 
   def dataOpt: Opts[js.Dynamic] =
     Opts
-      .option[String]("data", "Argument map for aqua function in JSON format", "d", "json")
+      .option[String]("data", "Arguments map in JSON format. Example: aqua run -d '{\"someArg1\": [1,2,3], \"someArg2\": 1}' -f 'someFunc(someArg1, someArg2)' -i aqua -a /dns4/stage.fluence.dev/tcp/19001/wss/p2p/12D3KooWHCJbJKGDfCgHSoCuK9q4STyRnVveqLoXAPBbXHTZx9Cv", "d", "json")
```

Comment:
- Why stage?
- Why docs just for the `--data` argument contains so much data? The full example should go to `aqua run`'s docs I suppose.

Reply:
Agree, examples usually represented as a separate section in man pages or help mesages.